libr/core/disasm.c

```diff
@@ -1070,7 +1073,7 @@ static void ds_build_op_str(RDisasmState *ds, bool print_color) {
 				core->parser->flagspace = NULL;
 			}
 		}
-		if (ds->analop.refptr) {
+		if (core->parser->relsub && ds->analop.refptr) {
```

Comment:
Arent we missing some references here? imho relsub shouldnt block refptr to work

Reply:
`relsub` is just a proxy for the config var `asm.relsub`, and not checking it will result in situations where `asm.relsub=0` but the substitution is still done. There is a test for this.